homeassistant/components/air_quality/__init__.py

```diff
@@ -13,43 +13,43 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_AIR_POLLUTANTS_AQI = 'air_quality_index'
-ATTR_AIR_POLLUTANTS_ATTRIBUTION = 'attribution'
-ATTR_AIR_POLLUTANTS_C02 = 'carbon_dioxide'
-ATTR_AIR_POLLUTANTS_CO = 'carbon_monoxide'
-ATTR_AIR_POLLUTANTS_N2O = 'nitrogen_oxide'
-ATTR_AIR_POLLUTANTS_NO = 'nitrogen_monoxide'
-ATTR_AIR_POLLUTANTS_NO2 = 'nitrogen_dioxide'
-ATTR_AIR_POLLUTANTS_OZONE = 'ozone'
-ATTR_AIR_POLLUTANTS_PM_0_1 = 'particulate_matter_0_1'
-ATTR_AIR_POLLUTANTS_PM_10 = 'particulate_matter_10'
-ATTR_AIR_POLLUTANTS_PM_2_5 = 'particulate_matter_2_5'
-ATTR_AIR_POLLUTANTS_SO2 = 'sulphur_dioxide'
-
-DOMAIN = 'air_pollutants'
+ATTR_AIR_QUALITY_AQI = 'air_quality_index'
```

Comment:
Why would the constant even have `AIR_QUALITY` in the name?

Reply:
Changing that to ATTR_AQI. 
Should I change the other constants?